### PR TITLE
Jobseeker notified job revoked 481

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#481

### CODE REVIEW

**Comment by vijayakarumudi** [comment]
Updated PR for review

File: app/views/job_seeker_mailer/job_revoked_jobseeker.html.haml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  This is to let you know that you had applied to the job
+  %strong
+  = link_to @job.title, job_url(id: @job.id)
+  on 
+  = @job_seeker.application_for_job(@job).created_at.strftime('%A, %B %d, %Y')
+%p
+  If you have any other questions about the use of PETS
+  then please contact the PETS administrator at:
+  = mail_to("#{ENV['NOTIFICATION_EMAIL']}", 'PETS Admin',
+         subject: 'Job revoked email notification')
```
Comment (patmbolger) [inline app/views/job_seeker_mailer/job_revoked_jobseeker.html.haml]: This email looks like this:
screen shot 2016-12-22 at 10 10 56 am

Please change to look like this:
screen shot 2016-12-22 at 10 22 55 am

This code should do it:
```
%p
  Hello #{@job_seeker.full_name(last_name_first: false)},
%p
  This message is from the administrator of the PETS system for
  the jobs agency:
  %strong
    #{@job.company.agencies[0].name}.
%p
This is to let you know that a job that you had applied to has been revoked
and thus is no longer active.
%p
You had applied to the job:
%strong
= link_to @job.title, job_url(id: @job.id)
on
= @job_seeker.application_for_job(@job).created_at.strftime('%A, %B %d, %Y') + '.'
%p
  If you have any questions about this email
  then please contact the PETS administrator at:
  = mail_to("#{ENV['NOTIFICATION_EMAIL']}", 'PETS Admin',
         subject: 'Job revoked email notification')
```

File: spec/mailers/job_seeker_mailer_spec.rb [hunk 2]
```diff
@@ -53,19 +50,46 @@
...
-
   end
 
+  describe 'Job revoked' do
+    let(:job) { FactoryGirl.create(:job) }
+    let(:job_seeker) { FactoryGirl.create(:job_seeker) }
+    let!(:resume)      { FactoryGirl.create(:resume, job_seeker: job_seeker) }
+    let!(:application) do
+      job.apply job_seeker
+    end
```
Comment (vijayakarumudi) [inline spec/mailers/job_seeker_mailer_spec.rb]: I removed the lines.

File: spec/mailers/job_seeker_mailer_spec.rb [hunk 3]
```diff
@@ -53,19 +50,46 @@
...
+
+    let(:mail) do
+      allow(Pusher).to receive(:trigger)
+      stub_cruncher_authenticate
+      stub_cruncher_job_create
+      JobSeekerMailer.job_revoked(job_seeker, job)
+    end
```
Comment (vijayakarumudi) [inline spec/mailers/job_seeker_mailer_spec.rb]: I updated file

File: app/views/job_seeker_mailer/job_revoked_jobseeker.html.haml (hunk 1 above)
Comment (vijayakarumudi) [inline app/views/job_seeker_mailer/job_revoked_jobseeker.html.haml]: I applied changes for this template.

**Comment by joaopapereira** [comment]
@vijayakarumudi can you please rebase this PR? Thanks

**Comment by vijayakarumudi** [comment]
Previously, I had resolved two merge conflicts on this PR. Yesterday I did not have any conflicts in this file. I would like to know how this conflict was introduced.

**Comment by patmbolger** [comment]
The latest merge conflict was introduced by https://github.com/AgileVentures/MetPlus_PETS/pull/466.

One file in this PR was affected by that PR.

**Comment by vijayakarumudi** [comment]
Updated PR.

**Pull request merged**